test/unit/models/test_oauth2.py

```diff
@@ -34,7 +34,7 @@ def test_should_encode_access_token(self, encode, env):
         create_access_token(data=data, expires_delta=expires_at)
 
         encode.assert_called_with(
-            data_to_encode, secret_key, algorithm=algorithm
+            data_to_encode, SECRET_KEY, algorithm=algorithm
```

Comment:
Aqui pode manter o nome da variável em caixa baixa.

```suggestion
            data_to_encode, secret_key, algorithm=algorithm
```